tests/libvmi/vmi.go

```diff
@@ -125,8 +126,15 @@ func WithSEV() Option {
 	}
 }
 
-func baseVmi(name string) *kvirtv1.VirtualMachineInstance {
-	vmi := kvirtv1.NewVMIReferenceFromNameWithNS("", name)
+// WithNamespace configures the namespace in which the VMI will be created.
+func WithNamespace(namespace string) Option {
+	return func(vmi *kvirtv1.VirtualMachineInstance) {
+		vmi.Namespace = namespace
+	}
+}
+
+func baseVmi(namespace, name string) *kvirtv1.VirtualMachineInstance {
+	vmi := kvirtv1.NewVMIReferenceFromNameWithNS(namespace, name)
```

Comment:
The namespace should not be set. Can you clarify why we need it?

Reply:
https://github.com/kubevirt/kubevirt/pull/7551#discussion_r849195120